packages/client/tests/functional/issues/9678/tests.ts

```diff
@@ -59,5 +59,9 @@ testMatrix.setupTestSuite(
         mongo - isolation levels are not supported
       `,
     },
+    skipProviderFlavor: {
+      from: ['js_neon', 'js_pg'],
+      reason: 'Error: could not serialize access due to concurrent update',
```

Comment:
Looks like we need to investigate this, it looks related to `$transaction()` with `isolationLevel` set to `Serializable`?